apps/prairielearn/src/pages/enroll/enroll.html.ts

```diff
@@ -60,162 +67,32 @@ export function Enroll({
                                 ? html`
                                     <button
                                       type="button"
-                                      class="btn btn-sm btn-info"
+                                      class="btn btn-sm btn-info js-add-course"
```

Comment:
Is this class used anywhere?

Reply:
Nope, this is left over from an earlier implementation where I was listening for click events on the button instead of open events. Removed in a1d39e1.